tox.cover.sh

```diff
@@ -31,6 +31,8 @@ cover () {
     min=98
   elif [ "$1" = "certbot_dns_dnsmadeeasy" ]; then
     min=99
+  elif [ "$1" = "certbot_dns_gehirn" ]; then
+    min=92
```

Comment:
My system only reports one line as missing coverage (line 57 of `dns_gehirn.py`), giving you 99% coverage. Does yours show that more lines are missing coverage? If so, which lines? If not, I think we should set min to 99% so that people who modify this plugin in the future don't unintentionally decrease code coverage.